http/vibe/http/proxy.d

```diff
@@ -196,20 +196,29 @@ HTTPServerRequestDelegateS reverseProxyRequest(HTTPReverseProxySettings settings
 /// ditto
 HTTPServerRequestDelegateS reverseProxyRequest(string destination_host, ushort destination_port)
 {
+	URL url;
+	url.schema = "http";
+	url.host = destination_host;
+	url.port = destination_port;
+	auto settings = new HTTPReverseProxySettings;
+	settings.destination = url;
+	return reverseProxyRequest(settings);
+}
+
+/// ditto
+HTTPServerRequestDelegateS reverseProxyRequest(URL destination)
+{
 	auto settings = new HTTPReverseProxySettings;
-	settings.destinationHost = destination_host;
-	settings.destinationPort = destination_port;
+	settings.destination = destination;
 	return reverseProxyRequest(settings);
 }
 
 /**
 	Provides advanced configuration facilities for reverse proxy servers.
 */
 final class HTTPReverseProxySettings {
-	/// The destination host to forward requests to
-	string destinationHost;
-	/// The destination port to forward requests to
-	ushort destinationPort;
+	/// The destination URL to forward requests to
+	URL destination;
```

Comment:
This should get some backwards-compatibility properties:
```
/// Scheduled for deprecation - use `destination.host` instead.
@property ushort destinationPort() const { return destination.port; }
/// ditto
@property void destinationPort(ushort port) { destination.port = port; }

/// Scheduled for deprecation - use `destination.host` instead.
@property string destinationHost() const { return destination.host; }
/// ditto
@property void destinationHost(string host) { destination.host = host; }
```

To make that fully work, there would also need to be a default value of `URL("http", "")` for `destination`.